### ISSUE TITLE
Scanning Ports

### ISSUE THREAD

**Issue by scottshotgg** [aceberg/WatchYourLAN#79]
Very cool application and just the perfect amount of simplistic/detailed that I was looking for.
I tried out the port scanning feature and didn't get anything. After looking at the logs I notice these error logs which are pretty reliably produced by clicking 'Scan host'. For reference the actual IP it should be scanning is `10.32.0.191` but it instead is backwards; `191.0.32.10`:
```shell
wyl-1  | 2024/04/01 09:52:54 ERROR: lookup 191.0.32.10.in-addr.arpa. on 127.0.0.53:53: no such host
wyl-1  | 2024/04/01 09:52:58 ERROR: strconv.Atoi: parsing "": invalid syntax
wyl-1  | 2024/04/01 09:52:58 ERROR: strconv.Atoi: parsing "": invalid syntax

```

**Comment by JSouthGB** [comment]
I'm seeing the same when attempting to scan for open ports.

**Comment by DobroFenix** [comment]
Yes, the functionality of scanning open ports on hosts would also be useful.
And so also receives an alert about the appearance of a new open port.
